File: examples/Python/valid.py
"""Valid Python sample."""

class Student:
    def __init__(self, name: str, age: int):
        self.name = name
        self.age = age

    def is_adult(self):
        return self.age >= 18


students = [Student("Ayesha", 20), Student("Ali", 16)]
for student in students:
    if student.is_adult():
        print(f"{student.name} is eligible")

File: examples/javascript/valid.js
class Student {
    constructor(name, age) {
        this.name = name;
        this.age = age;
    }

    isAdult() {
        return this.age >= 18;
    }
}

const student = new Student("Ayesha", 20);
if (student.isAdult()) {
    console.log(`${student.name} is eligible`);
}

